docs/docs/policies.mdx

```diff
@@ -689,36 +993,59 @@ default restart, label `2` a greeting and `3` indicates goodbye.
 
 #### Tracker Featurizers
 
-It's often useful to include a bit more history than just the current state
-when predicting an action. The `TrackerFeaturizer` iterates over tracker
-states and calls a `SingleStateFeaturizer` for each state to create numeric
-input features for a policy.
-The target labels correspond to bot actions or bot utterances
-represented as index in a list of all possible actions.
+A policy can be trained to learn two kinds of labels -
+
+1. The next most appropriate action to be triggered by the assistant. For example, [`TEDPolicy`](#ted-policy) is trained to do this.
+2. The next most likely intent that a user can express. For example, [`IntentTEDPolicy`](#intenttedpolicy) is trained to do this.
+
+Hence, a tracker can be featurized to learn one of the above kind of labels.
+Depending on the policy, the target labels correspond to bot actions or bot utterances
+represented as an index in a list of all possible actions or
+set of intents represented as an index in a list of all possible intents.
 
-There are two different tracker featurizers:
+There are three different tracker featurizers:
```

Comment:
"Tracker featurizers come in three different flavours:" (this avoids the "There are" at the beginning of the sentence, which carries no information)